### ISSUE TITLE
Calibration of a prepopulated image @Parameter does not update

### ISSUE THREAD

**Issue by rimadoma** [scijava/scijava-common#240]
The calibration info of a prepopulated image does not update. More specifically `CalibratedAxis.getScale` does not return the correct value.

Steps to reproduce:
1. Create a `ContextCommand` plugin that has a `@Parameter Dataset dataset` as input
2. Open "bat_cochlea_volume.tif" (_File > Open Samples > Bat Cochlea Volume_) in ImageJ. This is an uncalibrated image.
3. Debug run the plugin. Scale for each axis should be _1.0_
5. Calibrate "bat_cochlea_volume.tif" (_Analyze > Set Scale_)
6. Set options: _Distance in pixels_ to _1_, _Known distance_ to _1.00_, _Pixel aspect ratio_ to _0.5_, _Unit of length_ to _mm_
7. Rerun the plugin. Scale for dimension _1_ should be _0.5_

If you skip step 3 and run the plugin for the first time after setting the calibration, it is displayed correctly.

**Comment by ctrueden** [comment]
Yes, this is an IJ1<->IJ2 syncing issues. It would be fixed by an improved syncing/wrapping strategy, like so many other legacy bugs of this nature.
